testdata/scripts/health/health.txtar

```diff
@@ -0,0 +1,25 @@
+exec chainlink node start -p password -a creds &
+exec sleep 8
```

Comment:
Just curious about this value - feels sensible to get through all the node initialization? It does to me, but just wondering. Maybe a comment here for future readers would be helpful

Reply:
I would like to replace it with something that waits on the node to be ready, for example:
```
curl --retry 5 --retry-connrefused https://example.com
```

Reply:
This seems to work.